lib/busbar_cli/models/environment.rb

```diff
@@ -36,4 +38,12 @@ def pretty_settings
       "\t#{setting}: #{value}"
     end.join("\n")
   end
+
+  def pretty_components
+    components.map do |component, _|
+      component.map do |attribute, value|
```

Comment:
should we `.sort` by the component name (like how `pretty_settings` is sorted?) Don't feel terribly strongly about this, but may be nice to have "`click-web`, `clock`, `web`, `worker`, `viewability_worker`" in that order..what you do think?

Reply:
Good call... Let me give a look on this in a few! =))

Reply:
Done! =))